building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -1,6 +1,28 @@
 #!/bin/bash
-set -eu
+set -e -u
+rm -f resources/GIT_VERSION
+
+git update-index --refresh -q
+set +e
+git diff-index --quiet HEAD --
+GIT_DIFF_RETVAL=$?
+set -e
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+set +e
+u="$(git ls-files --exclude-standard --others "${GIT_ROOT}")" && test -z "$u"
+GIT_LS_RETVAL=$?
+set -e
```

Comment:
This seems rather complicated. Maybe something from https://unix.stackexchange.com/a/155077 might be easier?

Reply:
WiIl simplify.